apiaudio/logging.py

```diff
@@ -29,7 +29,7 @@ def format(self, record):
         return formatter.format(record)
 
 
-def SDKLogger(level=logging.WARNING):
+def SDKLogger(level="WARNING"):
```

Comment:
the old method is safer, since you are using 'type' rather than string

Reply:
yes, but I want to enable users to specify logging level without importing logging